frontend/src/components/VendorListOfOneMarket.js

```diff
@@ -2,10 +2,16 @@ import { Link } from 'react-router-dom';
 import VendorCard from '../components/VendorCard';
 
 const VendorListOfOneMarket = ({ market }) => {
+  const sortedList = market.vendors.sort(function (a, b) {
+    const nameA = a.name.toUpperCase();
```

Comment:
Ha máshol is fog a frontend rendezni, akkor érdemes a sort functiont kiszervezni.